revolv/payments/services.py

```diff
@@ -13,12 +15,14 @@ class PaymentServiceException(Exception):
 class PaymentService(object):
 
     @classmethod
-    def create_payment(cls, user, amount, payment_instrument):
+    def create_payment(cls, user, entrant, amount, project, payment_instrument):
```

Comment:
This method has a lot of arguments; do you think it's worth making a small class / named tuple to pass this stuff around?